packages/truffle-core/lib/testing/testrunner.js

```diff
@@ -124,90 +144,117 @@ TestRunner.prototype.resetState = function(callback) {
+      }
+    ],
+    function(err, result) {
+      if (err) return callback(err);
 
-    logs.forEach(function(log) {
-      var event = self.known_events[log.topics[0]];
+      var logs = result.result;
 
-      if (event == null) {
-        return;
+      if (logs.length == 0) {
+        self.logger.log("    > No events were emitted");
+        return callback();
       }
 
-      var types = event.abi_entry.inputs.map(function(input) {
-        return input.indexed == true ? null : input.type;
-      }).filter(function(type) {
-        return type != null;
-      });
+      self.logger.log("\n    Events emitted during test:");
+      self.logger.log("    ---------------------------");
+      self.logger.log("");
 
-      var values = abi.decodeLog(event.abi_entry.inputs, log.data, log.topics);
-      var index = 0;
+      logs.forEach(function(log) {
+        var event = self.known_events[log.topics[0]];
 
-      var line = "    " + event.abi_entry.name + "(";
-      line += event.abi_entry.inputs.map(function(input) {
-        var value;
-        if (input.indexed == true) {
-          value = "<indexed>";
-        } else {
-          value = values[index];
+        if (event == null) {
+          return;
         }
-        index += 1;
 
-        return input.name + ": " + `${value}`
-      }).join(", ");
-      line += ")";
-      self.logger.log(line);
-    });
-    self.logger.log(  "\n    ---------------------------");
-    callback();
-  });
+        var types = event.abi_entry.inputs
+          .map(function(input) {
+            return input.indexed == true ? null : input.type;
+          })
+          .filter(function(type) {
+            return type != null;
+          });
+
+        var values = abi.decodeLog(
+          event.abi_entry.inputs,
+          log.data,
+          log.topics
+        );
+        var index = 0;
+
+        var line = "    " + event.abi_entry.name + "(";
+        line += event.abi_entry.inputs
+          .map(function(input) {
+            var value;
+            if (input.indexed == true) {
+              value = "<indexed>";
+            } else {
+              value = values[index];
+            }
+            index += 1;
+
+            return input.name + ": " + `${value}`;
+          })
+          .join(", ");
+        line += ")";
+        self.logger.log(line);
+      });
+      self.logger.log("\n    ---------------------------");
+      callback();
+    }
+  );
 };
 
-TestRunner.prototype.snapshot = function(callback) {
+(TestRunner.prototype.snapshot = function(callback) {
   this.rpc("evm_snapshot", function(err, result) {
     if (err) return callback(err);
     callback(null, result.result);
   });
-},
-
-TestRunner.prototype.revert = function(snapshot_id, callback) {
-  this.rpc("evm_revert", [snapshot_id], callback);
-};
+}),
+  (TestRunner.prototype.revert = function(snapshot_id, callback) {
+    this.rpc("evm_revert", [snapshot_id], callback);
+  });
```

Comment:
Whoa what's up with this pair?

Reply:
ah, it was the comma on line 206 after the `snapshot` method.